doc/sphinxext/docscrape.py

```diff
@@ -428,7 +429,11 @@ def __init__(self, func, role='func'):
             func, func_name = self.get_func()
             try:
                 # try to read signature
-                argspec = inspect.getargspec(func)
+                if 3 > sys.version_info[0]:
+                    argspec = inspect.getargspec(func)
+                else:
+                    # getargspec deprecated in python3
+                    argspec = inspect.getargspec(func)
```

Comment:
There's something I'm missing here because the 2 lines are identical for python 2 and 3 cases:
```
argspec = inspect.getargspec(func)
```
It should be:
```
argspec = inspect.signature(func)
```
in the `else:`, right ?